packages/react/.eslintrc.js

```diff
@@ -0,0 +1,122 @@
+module.exports = {
+  env: { node: true },
+  root: true,
+  ignorePatterns: [
+    '.eslintrc.js',
+    'jest.config.js',
+    'tsup.config.ts',
+    'scripts',
+    'dist',
+    'node_modules',
+
+    // TODO Delete this file once the above directories are no longer ignored
```

Comment:
What does this comment mean?

Reply:
Kind of just took action on this but since the `plugin:react/recommended` rules include importing React in related files, and that we **sometimes** do it already, that we should delete _react-shim.js_ once the entire package is passing linting 😉

Reply:
I think we may not need the `react-shim.js` file at all.  My understanding is that importing `React` is no longer required unless it's actually used in the component:
https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html

Looks like we would need to add `plugin:react/jsx-runtime` according to docs: https://www.npmjs.com/package/eslint-plugin-react